lightly/data/_helpers.py

```diff
@@ -62,9 +60,8 @@ def _contains_subdirs(root: str):
 
     """
     with os.scandir(root) as scan_dir:
-        return any(
-            [f.is_dir() for f in scan_dir if not _is_lightly_output_dir(f.name)]
-        )
+        return any(f.is_dir() for f in scan_dir \
+            if not _is_lightly_output_dir(f.name))
```

Comment:
This line essentially does the following:
```
for f in scan_dir
    if not _is_lightly_output_dir(f.name):
        if f.is_dir:
            return True
return False
```
For a folder with many images, both if-conditions are checked every time. However, it could be made faster by reversing the order of the two if-conditions. It would cause the `_is_lightly_output_dir(f.name)` to be checked a maximum of two times.
Solution: `return any(f.is_dir() and not _is_lightly_output_dir(f.name) for fi in scan_dir)`